### ISSUE TITLE
Stray HR on Training Kit section on the home page

### ISSUE THREAD

**Issue by jimmylogan** [PanicInitiative/panicbutton.io#19]
On the [Training Kit](https://panicbutton.io/#pact) section on the home page, the final sentence states:

```
"You can browse the cards online, or print them for use in workshops.<hr>"
```

The ```<hr>``` should be removed.

**Comment by jmatsushita** [comment]
Probably the same issue than #10 solved with dc3690c5eb1c4a848e3aa510efafe54ee7ff3fbe

**Comment by naturalshine** [comment]
@jimmylogan this is fixed & closed by this [pull merge](https://github.com/PanicInitiative/panicbutton.io/commit/8a8ec2561c43ccb43ab231906c578a2a53ab9ad9)
